app/Http/Controllers/MismatchController.php

```diff
@@ -22,13 +22,13 @@ public function index(MismatchesRequest $request)
 
         // limit to 'pending',
         // unless include_reviewed parameter is provided
-        if (!$request->include_reviewed) {
+        if ($request->include_reviewed != 'true') {
```

Comment:
oh my, I can't believe I missed that :man_facepalming:

Reply:
Non blocking, but this looks like another - more robust way of handling boolean query params: https://laravel.com/docs/8.x/requests#retrieving-boolean-input-values